### PR TITLE
Adding a simple test

### PR DESCRIPTION
**Pull request by jprabhas**
This is a test PR to try out PR workflow on openxla/xla

### CODE REVIEW

**Comment by tpopp** [comment]
Hello, just to confirm, are you able to see the failing test and is it expected?

**Comment by puneith** [comment]
@jprabhas you should be able to see ResultStore link with failing tests. Can you confirm?

File: xla/service/gpu/gpu_layout_assignment_test.cc [hunk 1]
```diff
@@ -380,6 +380,32 @@ TEST_F(LayoutAssignmentTest, ConvCuDNNBF16) {
...
+  }
+
+  const char* hlo = R"(
+
+  HloModule jit_conv_general_dilated
+
+  ENTRY main.4 {
+    Arg_0.1 = f16[1,64,64,16]{3,2,1,0} parameter(0), sharding={replicated}
+    Arg_1.2 = f16[3,3,16,32]{3,2,1,0} parameter(1), sharding={replicated}
+    ROOT convolution.3 = f16[1,64,64,32]{3,2,1,0} convolution(Arg_0.1, Arg_1.2), window={size=3x3 pad=1_1x1_1}, dim_labels=b01f_01io->b01f, metadata={op_name="jit(conv_general_dilated)/jit(main)/conv_general_dilated[window_strides=(1, 1) padding=((1, 1), (1, 1)) lhs_dilation=(1, 1) rhs_dilation=(1, 1) dimension_numbers=ConvDimensionNumbers(lhs_spec=(0, 3, 1, 2), rhs_spec=(3, 2, 0, 1), out_spec=(0, 3, 1, 2)) feature_group_count=1 batch_group_count=1 lhs_shape=(1, 64, 64, 16) rhs_shape=(3, 3, 16, 32) precision=None preferred_element_type=None]" source_file="/usr/local/lib/python3.8/dist-packages/flax/linen/linear.py" source_line=438}
```
Comment (SandSnip3r) [inline xla/service/gpu/gpu_layout_assignment_test.cc:397]: It's probably better to exclude metadata from a test unless the test is testing for that specifically.

File: xla/service/gpu/gpu_layout_assignment_test.cc [hunk 2]
```diff
@@ -380,6 +380,32 @@ TEST_F(LayoutAssignmentTest, ConvCuDNNBF16) {
...
+  }
+)";
+
+  MatchOptimizedHlo(hlo, R"(
+  // CHECK: [[P0:%[^ ]+]] = f16[1,64,64,16]{3,2,1,0} parameter(0), sharding={replicated}
```
Comment (SandSnip3r) [inline xla/service/gpu/gpu_layout_assignment_test.cc:402]: Does FileCheck work when these `CHECK` lines are prefixed with this comment `//`?

Comment (jprabhas) [reply xla/service/gpu/gpu_layout_assignment_test.cc:402]: Yes, it does. I see several examples in the code base as well https://github.com/tensorflow/tensorflow/blob/ac80c38663a9e645c28152c411c55ce99c5ea858/tensorflow/compiler/xla/service/gpu/conv_layout_normalization_test.cc#L73

**Review by SandSnip3r — APPROVED** [review]

**Review by ddunl — APPROVED** [review]
Awesome, thanks for the change, and thank you Victor for your review!

**Pull request closed without merging**